### PR TITLE
Feature/error page

### LINKED ISSUE #8: FE - <UrlError>

### PR DESCRIPTION
**Pull request by judy0ye**
closes #8 

What I did: Add error component

Testing Notes/Extras: 

Did this break anything?
- [x]  No
- [ ]  Yes

Type of change
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  The code is DRY. If it's not, I tried my best
- [ ]  I reviewed my code before pushing

What's next? Router

### CODE REVIEW

**Review by aliceabarca — APPROVED** [review]
amazing job with the error handling!

File: src/App.js
Comment (apete12) [inline src/App.js:1]: Nice job adding the error state!

**Review by apete12 — APPROVED** [review]

File: src/components/Error/Error.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import './Error.css'
+import shockedCat from '../../images/hang-niu-Tn8DLxwuDMA-unsplash.jpg'
+
+function Error({error}) {
+  return (
+    <section className='error'>
+      <h2>{error}</h2>
+      <img src={shockedCat}/>
```
Comment (paulina-isabel) [inline src/components/Error/Error.js:8]: We should probs give an alt text to this image for accessibility!

**Review by paulina-isabel — APPROVED** [review]
Love the interpolation of error.message and addition of an error state. Great job!

File: src/components/Error/Error.js (hunk 1 above)
Comment (apete12) [reply src/components/Error/Error.js:8]: Good catch!

**Pull request merged**